### ISSUE TITLE
[Bug] Crouchy Zoomies

### ISSUE THREAD

**Issue by BoB-Client** [Thy-Vipe/BeastsOfBermuda-issues#9952]
_Originally written by **Grammar Patrol | 76561198979288831**_
Game Version: 1.1.1381

*===== System Specs =====
CPU Brand: 11th Gen Intel(R) Core(TM) i3-1115G4 @ 3.00GHz
Vendor: GenuineIntel
GPU Brand: Intel(R) UHD Graphics
GPU Driver Info: Unknown
Num CPU Cores: 2
===================*
Context: **Anything other than Coah, Sai, Mosa, Krono, Elasmo, Pt, and Tropeo** 
Map: Rival_Shores

While crouching on any dino (Or for apato backing up) as a juvinile/hatchling the animation is..... BROKENLY sped up. Looks like you're going 200 miles an hour when you're going at an ants pace

**Comment by Undercooked-Mothman** [comment]
i dont think this can be changed without making those baby's anims be very very slidey
